docs/releases/0_9_14.rst

```diff
@@ -172,15 +172,15 @@ Carbon
 * Unable to load ``AGGREGATION_RULES`` (drawks)
 * Compatibility with Twisted 13.2.0 (esc, drawks)
 * Incorrect log rotation documentation (mleinart)
-* Fix carbon-cache cpu usage 100% when sent metric with too big name (jssr, deniszh)
-* Fix aggregator replication factor setting (jssr, deniszh)
+* Fix carbon-cache cpu usage 100% when sent metric with too big name (jssjr, deniszh)
+* Fix aggregator replication factor setting (jssjr, deniszh)
 * Change the max update on shutdown (f80)
 * Document the fact that one can use regexps in the aggregation-rules (ctavan)
-* Move tests to tox (jssr)
-* Add hup signal handler (jssr)
-* Fix instrumentation (avishai-ish-shalom, jssr)
+* Move tests to tox (jssjr)
+* Add hup signal handler (jssjr)
+* Fix instrumentation (avishai-ish-shalom, jssjr)
 * Fix exception handling (steve-dave)
-* Fix CACHE_WRITE_STRATEGY (jssr)
+* Fix CACHE_WRITE_STRATEGY (jssjr)
 * Fix aggregated metrics (pgul, ctavan)
 * Logging fixes (obfuscurity, piotr1212)
 * Fix race condition for full queues (mleinart)
```

Comment:
Protocols and clients are now "pluginized", which will allow custom ones to be writtens.